lib/editor/util/timetable.js

```diff
@@ -40,12 +40,20 @@ export function getHeaderColumns (
   return columns.filter(c => c.type !== 'DEPARTURE_TIME')
 }
 
-export function parseTime (timeString: string) {
-  const date = moment().startOf('day').format('YYYY-MM-DD')
-  return moment(date + 'T' + timeString, TIMETABLE_FORMATS).diff(
-    date,
-    'seconds'
-  )
+/**
+* Handles pasted data from clipboard (e.g. from CSV file)
+* If departure/arrival time cell, pastes in time format, otherwise returns string as is
+ */
+export function parseCellValue (timeString: string, col: TimetableColumn) {
```

Comment:
watch out this may PR may include changes from other prs

Reply:
Thanks for catching that! I had merged your PR for some local work I was doing. I'll revert that

Reply:
This was reverted by https://github.com/ibi-group/datatools-ui/pull/984/commits/0036f0ce1e18b423fb02c37bfbc44f0d3472ce59